src/pyclvm/instance/start.py

```diff
@@ -70,8 +70,7 @@ def _is_stopped_or_terminated(
     instance: Union[Ec2InstanceProxy, GcpInstanceProxy, AzureInstanceProxy],
 ) -> None:
     print(f"Starting {instance_name} ...")
-    instance.start()
-    print(f"{instance_name} is running")
+    instance.start(wait=False)
```

Comment:
Why the `print(f"{instance_name} is running")` line was removed?

Reply:
Logically it does not make sense. If a VM has "stopped" status and then begins to run, the status changes to transition state "starting/running", i.e "begin to start/run", but the VM does'n not ready to work. 
f"{instance_name} is running") could confuse a user.
the same logic with stop/stopping/stopped statuses.